packages/block-library/src/query/edit/inspector-controls/order-control.js

```diff
@@ -23,6 +23,16 @@ const orderOptions = [
 		label: __( 'Z → A' ),
 		value: 'title/desc',
 	},
+	{
```

Comment:
These options should be available only to post types that support page attributes. Where and how do we add that logic? Here is a reference to how the "order" setting is decided at the page settings three-dots:

https://github.com/WordPress/gutenberg/blob/be5ce8aefa49b394a57aa540d9c9b091510ed8f1/packages/editor/src/dataviews/store/private-actions.ts#L156-L158

Reply:
I don't know if it helps but there is a `PageAttributesCheck`, that you can take inspiration from:
https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/page-attributes/check.js

Reply:
Can usePostTypes not be updated for this purpose? https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query/utils.js#L101

Reply:
This has been implemented using an approach similar to the `PageAttributesCheck` suggsted here. Thank you @carolinan!